prefect_azure/container_instance.py

```diff
@@ -214,6 +216,12 @@ class AzureContainerInstanceJob(Infrastructure):
             "defaults to a Prefect base image matching your local versions."
         ),
     )
+    name: Optional[str] = Field(
+        default=None,
+        description=(
+            "Name of the Azure Container Instance."
+        ),
+    )
```

Comment:
The `Infrastructure` class that `AzureContainerInstanceJob` subclasses already has a `name` attribute, so adding `name` on this classes shouldn't be necessary.
```suggestion
```